src/main/java/com/CarRentalAgency/entity/Car.java

```diff
@@ -63,6 +63,14 @@ public class Car {
     @JoinColumn(name = "dealer_id", nullable = false)
     private Dealer dealer;
 
+    public void setKilometre(int i) {
+        this.kilometres = i;
+    }
+
+    public void setCarName(String test) {
+        this.name = test;
+    }
```

Comment:
**Suggestion:** Rename the method setCarName to setName for consistency with the variable it's setting.
```suggestion
    public void setName(String test) {
        this.name = test;
    }
```